tests/test_FileSR.py

```diff
@@ -438,6 +439,9 @@ class TestShareFileSR(unittest.TestCase):
     ERROR_524 = "Unknown error 524"
 
     def setUp(self):
+        xml_patcher = mock.patch('sm.core.xs_errors.XML_DEFS','libs/sm/core/XE_SR_ERRORCODES.xml')
```

Comment:
If we have this mocked at the class level this bit in `setUp` should not be required?

Reply:
This one wasn't mocked at the class level. It is now though.